docs/UserGuide.md

```diff
@@ -103,16 +125,16 @@ Let us get started!
 
     
 
-### **Notes about the command format:**<br>
+### **Notes about the command format:**
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  > `delete id/NUSID`, `NUSID` is a parameter which need to be supplied.
+  > In the delete command as such: `delete id/NUSID`, `NUSID` is a parameter which need to be supplied.
 
 * Items in square brackets are optional.<br>
   > `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or simply as `n/John Doe` without specifying the `TAG` as it is optional.
 
 * Items with `…`​ after them can be used multiple times or zero times.<br>
```

Comment:
Can you update the multiplicity to one or more as for the `group id/NUSID...` allows multiple people to be grouped but need at least one NUSID provided?